docs/articles/sdk/configure-qtcreator.rst

```diff
@@ -0,0 +1,52 @@
+Configuring QtCreator to use the SDK
+====================================
+
+After the installation of the SDK and sourcing environment, there are couple of
+steps to do for running QtCreator in target environment. This roadmap assumes
+that SDK and QtCreator should be installed and environments are sourced already.
+
+Adding a new kit
+--------------------
+
+To be able to cross-compile via QtCreator, new build kit should be added to
+QtCreator.
+
+* Open QtCreator
+* Open Options form from Tools->Options, then click ``Build & Run`` item on the
+left panel of option window.
+* Select ``Kits`` tab and click ``Add`` button.
+* Name your new kit and choose ``Device type`` as ``GenericLinuxOsType``.
+* Then browse and submit the sysroot. If you don't know sysroot location you can
+discover by typing ``echo ${SDKTARGETSYSROOT}`` to console.
+* Then choose C compiler. If cross compiler of the SDK is not listed, it should
+be added as a new gcc. To add new gcc click ``Manage`` button next to compiler.
+Add you GCC compiler path here. You can discover this path from environment as well.
+Just type ``which ${CC}`` to console to discover.
+* Choose C++ compiler as you did in previous step. You can discover path by typing
```

Comment:
"Choose the C++ compiler as you did the the previous step. You can find the path running"